### PR TITLE
DEVX-1426: update text to include EMIT CHANGES

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -287,7 +287,7 @@ In this demo, KSQL is authenticated and authorized to connect to the secured Kaf
    .. figure:: images/ksql_queries_list.png
       :alt: image
 
-8. View messages from different KSQL streams and tables. Click on your stream of choice and select **Query** to open the Query Editor. The editor shows a pre-populated query, like ``select * from WIKIPEDIA;``, and it shows results for newly arriving data.
+8. View messages from different KSQL streams and tables. Click on your stream of choice and select **Query** to open the Query Editor. The editor shows a pre-populated query, like ``select * from WIKIPEDIA emit changes;``, and it shows results for newly arriving data.
```
Comment (rspurgeon) [inline docs/index.rst]: Should we strive to standarize on upper case ksql keyword syntax?

Comment (ybyzek) [reply docs/index.rst]: Scope is bigger than this PR.  (following up in separate conversation)

Comment (rspurgeon) [reply docs/index.rst]: I see the connection to c3 in the doc snippet now

Comment (rspurgeon) [reply docs/index.rst]: I see the connection to c3 in the doc snippet now

**Pull request merged**